jsgtk_modules/fs.js

```diff
@@ -150,8 +157,12 @@ const
       mainloop.wait();
     },
     readFileSync: function readFileSync(file, options) {
-      // TODO: supports options
-      return GFile.new_for_path(file).load_contents(null)[1];
+      // TODO: support the 'flag' option
+      let encoding = options && options.encoding || options;
```

Comment:
yup, same exact issue. If there is an options but no `encoding` you'll call `data.toString(object)` instead.

```js
let encoding = 1 < arguments.length ? (options.encoding || '') : '';

// ...

return encoding.length ? data.toString(encoding) : data;
```